### ISSUE TITLE
Enable Chat option

### ISSUE THREAD

**Issue by Ponnusamy1-V** [frappe/chat#42]
Turning off the Enable Chat option in chat settings also allows the user to send message if the chat icon is visible.

**Comment by mrPauwHaan** [comment]
yeah, we have the same issue. If it is disabled, the "chat us" option is still visible on the website.
